README.md

```diff
@@ -1,10 +1,31 @@
-### What is Kubernetes Iteration Toolkit?
+# What is Kubernetes Iteration Toolkit?
 
-Kubernetes Iteration Toolkit (or KIT) is a set of decoupled tools to accelerate the development of Kubernetes through testing. It combines a variety of open source projects to define an opinionated way to rapidly configure and test Kubernetes components on AWS.
+## What is KIT?
 
+[KIT](https://github.com/awslabs/kubernetes-iteration-toolkit) is a set of decoupled tools designed to accelerate the development of Kubernetes through testing. It combines a variety of open source projects to define an opinionated way to rapidly configure and test Kubernetes components on AWS.
 
-It consists of the following components at the moment.
+## Why did we build KIT?
 
-* [Operator](operator/README.md)
-* [KIT-environment](substrate/README.md)
-* [TestBed](testbed/README.md)
+The EKS Scalability team is responsible for improving performance across the Kubernetes stack. We started our journey by manually running tests against modified EKS dev clusters. This helped us to identify some bottlenecks, but results were difficult to demonstrate and reproduce. We wanted to increase the velocity of our discoveries, as well as our confidence in our results. We set out to build automation to help us configure cluster components, execute well known test workloads, and analyze the results. This evolved into KIT, and we’re ready to share it to help accelerate testing in other teams.
```

Comment:
I'm not sure if we should provide this section in the public repo, or we might want to consider removing the section talking about the difficulty of testing on EKS dev clusters.